### PR TITLE
Hello React Front End App

### PR DESCRIPTION
**Pull request by jugosack**
### In this pull request is completed the following:

**Features**

- [x] Create a new React app called 'hello-react-front-end'
- [x]  Initialize your project with Git.
- [x] Add linters
- [x] Create the App component with react-router.
- [x] Create the Greeting component that will display a greeting. Set it up as a route in your App component.
- [x] Create a store, an action and a reducer that will connect to you API endpont to get the random greeting.
- [x] Display the random greeting in your Greeting component.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable import/extensions */
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
```
Comment (Munsa1) [inline src/App.js]: - [ ] Kindly emove eslint-disable as linters help us write code in a standardized and approved manner

File: src/pages/Greeting.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable import/extensions */
```
Comment (Munsa1) [inline src/pages/Greeting.js]: - [ ] Kindly emove eslint-disable as linters help us write code in a standardized and approved manner

File: src/redux/greetingSlice.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (Munsa1) [inline src/redux/greetingSlice.js]: - [ ] Kindly emove eslint-disable as linters help us write code in a standardized and approved manner

File: src/redux/store.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable import/no-self-import */
```
Comment (Munsa1) [inline src/redux/store.js]: - [ ] Kindly emove eslint-disable as linters help us write code in a standardized and approved manner

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @jugosack, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Added A `Greet` Component :heavy_check_mark: 
Used React Router Dom :heavy_check_mark: 
Added An Action, Store And A Reducer :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @jugosack    :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**